### ISSUE TITLE
Supported MIME types - Is application/xml supported?

### ISSUE THREAD

**Issue by renmak** [BigstickCarpet/swagger-server#16]
Hello there,
I am trying to get mock response in XML format. 
- I have specified consumes and produces to -application/xml at top of yaml file. 
- Request includes content-type and accepts in header as application/xml. 

However i get response in JSON format. I am seeing following message in console. 
WARNING! GET /cars produces the MIME types that are not supported (application/xml). Using "json" instead

**Comment by winniehell** [comment]
This seems to be an issues of [swagger-express-middleware]. [Arrays and objects][call-of-sendObject] are always sent [as JSON][call-of-setContentType].

[swagger-express-middleware]: https://github.com/BigstickCarpet/swagger-express-middleware
[call-of-sendObject]: https://github.com/BigstickCarpet/swagger-express-middleware/blob/eb417f90d211cbcf9864345cfebbb9c1ec84fd45/lib/mock/index.js#L206-L209
[call-of-setContentType]: https://github.com/BigstickCarpet/swagger-express-middleware/blob/eb417f90d211cbcf9864345cfebbb9c1ec84fd45/lib/mock/index.js#L235

**Comment by BigstickCarpet** [comment]
Yeah, this is a big item on the "to-do" list for Swagger Express Middleware.  I just don't have the time to implement it right now.  Busy as hell at work :(
